src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/AbstractMemberWriter.java

```diff
@@ -400,8 +400,8 @@ public Content getSummaryTableTree(TypeElement tElement) {
     }
 
     @Override
-    public Content getMemberTree(Content memberTree) {
-        return writer.getMemberTree(memberTree);
+    public Content getMember(Content memberContent) {
+        return writer.getMember(memberContent);
```

Comment:
As a comment on the underlying API (not your changes), this method is confusing or under-specified or both.   

I think this method name is too short.  In an `AbstractMemberWriter`, the member might reasonably be expected to be the underlying `Element`.